pkg/core/envs/envs.go

```diff
@@ -0,0 +1,33 @@
+package envs
+
+import (
+	"encoding/csv"
+	"errors"
+	"io"
+	"strings"
+)
+
+var (
+	// ErrInvalidCSVFormat when the number of columns is different than two
+	ErrInvalidCSVFormat = errors.New("Invalid csv format, expecting: key, value")
+)
+
+// ReadEnvs reads data from csv file to save it as a map for creating a secret
+func ReadEnvs(envVars string) (map[string]string, error) {
```

Comment:
Same as with `Resources` - this is very specific piece of functionality used by only one piece of the app (backend), so IMO it should reside there. Core should be something general-purpose, reused in many parts of the app (like rqctx)